### PR TITLE
tech(module): add shared module

### LINKED ISSUE #9: Shared Module - app/shared/***
Services must not be defined here. 
1. Header #20 
1. Mainmenu
1. Footer #21 

```
ng g m shared
```

### PR DESCRIPTION
**Pull request by shangan23**
## Issue ID : #9
Fixes #9 

## Summary

Creating shared modules allows you to organize and streamline your code. You can put commonly used directives, pipes, and components into one module and then import just that module wherever you need it in other parts of your app.

## Steps to do this Tech Update

- Angular CLI - `ng g m shared`
  - Creates - `CREATE src/app/shared/shared.module.ts (192 bytes)`

## Checklist

- [x] Have you added the summary of what your changes do and why you'd like us to include them?
- [x] Have you added the steps you followed?
- [ ] Have the docs been updated to match the changes in the PR?
- [ ] Have the files been linted and formatted?
- [ ] Have the tests been updated to match the changes in the PR?
- [ ] Have you run the tests locally to confirm they pass?

### What is the need for this tech update, and the steps to reproduce the issue?
Best practice 

### What is the current status?
Created a module

### How does this PR fix the problem, Screenshot Please?
Not related to UI, it simply creates a file and folder `CREATE src/app/shared/shared.module.ts (192 bytes)`

### CODE REVIEW

**Review by karthikeyanpa12 — APPROVED** [review]

**Review by ramsunvtech — APPROVED** [review]

File: src/app/shared/shared.module.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
+
```
Comment (ramsunvtech) [inline src/app/shared/shared.module.ts:5]: please keep single line instead of 3 empty new lines

**Comment by ramsunvtech** [comment]
@shangan23 commit message can be like below.

```
tech(shared/module): add base code
```

File: src/app/shared/shared.module.ts (hunk 1 above)
Comment (naveenvignesh5) [reply src/app/shared/shared.module.ts:5]: @shangan23 any update on this ?

Comment (naveenvignesh5) [reply src/app/shared/shared.module.ts:5]: Approving this. Kindly fix this in the next push.

**Review by naveenvignesh5 — APPROVED** [review]

**Review by jpukg — APPROVED** [review]

**Pull request merged**